src/socket.rs

```diff
@@ -48,6 +53,8 @@ pub enum GeneralTimestampMode {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Default)]
 pub enum InterfaceTimestampMode {
+    HardwareAll,
+    HardwareRecv,
     HardwarePTPAll,
```

Comment:
is there some documentation that we can link to here?

Reply:
How do you mean? These are internal constants, not something the OS defines